rails-commenting.rb

```diff
@@ -48,15 +49,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8) If the blog post is successfully deleted, the user is redirected to the index action of the blog posts.
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # ---9) The private keyword denotes that the following methods are private and can only be accessed within the BlogPostsController class.
   private
   def blog_post_params
-    # ---10)
+    # ---10) The blog_post_params method is a private method that filters and permits only the :title and :content parameters from the form submissions, which helps protect against unauthorized or malicious data being passed into the application. This method is used in the create and update actions to ensure that only the allowed parameters are used to create or update a blog post.
```

Comment:
I'm trying to come up with some constructive criticism, but you won't let me!  Well done! ⭐️